.expeditor/scripts/bk_linux_exec.sh

```diff
@@ -31,8 +31,10 @@ echo "--- Config information"
 
 echo "!!!! RUBY VERSION !!!!"
 ruby --version
-echo "!!!! BUNDLE LOCATION !!!!"
-which bundle
+echo "!!!! BUNDLER LOCATION !!!!"
+which bundler
+echo "!!!! BUNDLER VERSION !!!!"
+bundler -v
```

Comment:
`bundle -v`?

Reply:
it's an alias, but I updated the 3 places I used bundler not bundle